lib/mercury/fake.rb

```diff
@@ -17,15 +17,15 @@
 # broken sockets, etc.
 class Mercury
   class Fake
-    def self.install(rspec_context, domain=:default)
+    def self.install(rspec_context, domain = :default)
```

Comment:
I think symbols are one of those examples where it's more readable with the space.